src/data/businessCase.ts

```diff
@@ -86,9 +86,14 @@ type lifecycleCostLinesType = {
   B: EstimatedLifecycleCostLines;
 };
 
-// Function that lets us know if a user has filled out an alternative B
-export const hasAlternativeB = (alternativeB: ProposedBusinessCaseSolution) => {
-  if (!alternativeB) {
+/**
+ * This function tells us whether the parameter alternativeSolution has been started
+ * @param alternativeSolution - an alternative solution case (e.g. A, B)
+ */
+export const hasAlternativeSolution = (
```

Comment:
Can we call this `alternativeSolutionHasFilledFields` (or something like that) so that the name is more clear? It feels weird to have a function called `hasAlternativeSolution` and then call it with... an alternative solution.